packages/specification/schemas/components/MedicationDispense/MedicationDispense.yaml

```diff
@@ -212,10 +212,41 @@ properties:
     items:
       type: object
       properties:
+        sequence:
+          type: integer
+          format: int32
+          example: 1
         text:
           type: string
           description: The dosage instructions expressed as text.
```

Comment:
Need to clarify that multiple dose text strings will be concatenated into the final prescription.

Reply:
Added to text description